### PR TITLE
Add a new page inviting contributions to Launchpad documentation

### PR DESCRIPTION
**Pull request by sn3rd**

### CODE REVIEW

File: .sphinx/spellingcheck.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ matrix:
...
...ontribute-to-our-docs/index.html|!_build/how-to/database-setup/index.html|!_build/how-to/submitting-a-patch/index.html|!_build/how-to/database-schema-changes-process/index.html|!_build/how-to/triage-bugs/index.html|!_build/explanation/registry/index.html|!_build/explanation/css-sprites/index.html|!_build/explanation/working-with-db-devel/index.html|!_build/explanation/pre-merge-reviews/index.html|!_build/explanation/live-patching/index.html|!_build/explanation/css/index.html|!_build/explanation/javascript-unittesting/index.html|!_build/explanation/engineering-overview-translations/index.html|!_build/explanation/testing/index.html|!_build/explanation/feature-flags/index.html|!_build/explanation/launchpad-ppa/index.html|!_build/explanation/branches/index.html|!_build/explanation/code/index.html|!_build/explanation/security-policy/index.html|!_build/explanation/database-performance/index.html|!_build/explanation/url-traversal/index.html|!_build/explanation/navigation-menus/index.html|!_build/explanation/storm-migration-guide/index.html|!_build/explanation/mail/index.html|!_build/explanation/javascript-buildsystem/index.html|!_build/explanation/javascript-integration-testing/index.html
```
Comment (jugmac00) [inline .sphinx/spellingcheck.yaml:12]: Excluding some converted pages from spellchecking was a means to an end to get the migration done in time.

We should not make it a habit to exclude newly written pages.

You can add new words to the `.custom_wordlist.txt` configuration file.

File: contribute-to-our-docs.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+**We warmly welcome community contributions, suggestions, fixes, and constructive feedback.**
+
+If you feel something is unclear, wrong, or broken, please don't hesitate to reach out to us by any of these means:
+
+- Join the `#launchpad IRC channel`_ at `irc.libera.chat`_ and ask your
+  questions.
+- Ask a question: https://answers.launchpad.net/launchpad
+- Send an e-mail to feedback@launchpad.net
+
+Or go ahead and submit an issue or pull request at https://github.com/canonical/launchpad-manual.
```
Comment (jugmac00) [inline contribute-to-our-docs.rst:15]: Personally, I would prefer to have people create issues in the very first place, as this feels more coherent, as the documentation is also hosted here on GitHub.

File: contribute-to-our-docs.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Feel free to suggest changes to any documentation topic with updated or more insightful information. The navigation structure, style, and content of our documentation follows the `Diátaxis`_ systematic framework for technical documentation authoring. This splits pages into tutorials, how-to guides, reference material, and explanatory text.
+
+Style and language
+++++++++++++++++++
+
+Launchpad's documentation is aimed at helping both beginners and experts. As such, we try to use appropriate writing for the subject, to use inclusive language, and to assume that the reader has as little prior knowledge as possible.
+
+Some general tips:
+
+- use a spell checker
```
Comment (jugmac00) [inline contribute-to-our-docs.rst:26]: We have a linter for that, so at one point we probably should suggest to use tooling, though I do not want anybody to be kept off from contributing when they do not feel comfortable to use make files.

File: contribute-to-our-docs.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- resist being overly formal
+- resist being overly verbose
+- verify links and examples
```
Comment (jugmac00) [inline contribute-to-our-docs.rst:29]: Maybe we could add a line that people "could" use make file to ensure high standards.

**Review by jugmac00 — APPROVED** [review]
Thanks a lot! This looks great!!!

A couple of comments, mostly inline.

Usually, we wrap the text after ~80 characters, which make it easier to read and manage, even when you could emulate that in the editor with "Word Wrap" or similar functions.

This is afaik neither documented nor enforced by linters, which we def. should do.

The link to this new guide on the left is easily overlooked.

I would suggest to additionally update https://github.com/canonical/launchpad-manual/blob/main/index.rst#project-and-community section, where we already have a contribution link for code, e.g. "Contribute to code / Contribute to documentation." - and make both terms a link.

File: .sphinx/spellingcheck.yaml (hunk 1 above)
Comment (sn3rd) [reply .sphinx/spellingcheck.yaml:12]: I actually did this, and it didn't work as I expected, so I ended up adding the exclusion in order to move on. I think we should have a separate exercise to fix this up.

File: contribute-to-our-docs.rst (hunk 2 above)
Comment (sn3rd) [reply contribute-to-our-docs.rst:15]: Good point; will amend.

**Pull request merged**